.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
no, by-definition ``.pxi`` are auto-generated, we *never* want to add them to the repo, while ``.pxi.in`` we always do.